sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -890,6 +895,7 @@ export class ManagementClient extends LinkEntity {
         }
       };
       request.application_properties![Constants.trackingId] = generate_uuid();
+      const associatedLinkName = this._getAssociatedReceiverName(this._context);
```

Comment:
Shouldnt we pass sessionId to `_getAssociatedReceiverName` here?

Reply:
Updated